spec/acceptance/docker_basic_crud_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'spec_helper_acceptance'
+
+describe 'docker CRUD tests' do
```

Comment:
This should describe what you're testing, so in this case the docker Puppet class.